src/pad_constant/problem_description.cpp

```diff
@@ -0,0 +1,46 @@
+/*******************************************************************************
+ *
+ * MIT License
+ *
+ * Copyright (c) 2024 Advanced Micro Devices, Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in all
+ * copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ *
+ *******************************************************************************/
+
+#include "miopen/pad_constant/problem_description.hpp"
+#include "miopen/names.hpp"
+
+namespace miopen {
+namespace pad_constant_fwd_contiguous {
+    NetworkConfig ProblemDescription::MakeNetworkConfig() const {
+        // Should be the only thing we need? (After all paddings are all the same operation kind)
+        auto dtype = xDesc.GetType();
+
+        std::ostringstream ss;
+        ss << "dtype" << dtype;
+
+        // no. i blame whoever designed this. why does the cache have launch configs.
+        ss << "xDesc" << xDesc;
```

Comment:
I think dtype and output_size will be sufficient.

Reply:
I've updated it to use `y_desc.GetElementSize()` instead. Running the unit tests seems to suggest that is the correct one for it to pass, so I'm leaving it as that. If there are introduced bugs, please mention it here so I can revert it to `yDesc`